### ISSUE TITLE
Inline chat: extra margin/padding using text button style

### ISSUE THREAD

**Issue by daviddossett** [microsoft/vscode#216770]
Current:

https://github.com/user-attachments/assets/7442b380-37fe-4b83-be97-e59c2294d816

Desired:

https://github.com/user-attachments/assets/715ab54d-d047-4bd8-b547-11d34b04fb98

Couldn't figure out how to get this without removing code that looks necessary for the current form button layout: https://github.com/microsoft/vscode/blob/94e56caf84aba117ad742033aa9f927b479f63b5/src/vs/workbench/contrib/inlineChat/browser/media/inlineChat.css#L71

**Comment by jrieken** [comment]
Yeah, the extra spacing is needed for button mode. When focused they have a rather large focus border. We might not need this when going with coloured buttons instead

**Comment by jrieken** [comment]
confident that this is now better
